test/unit/media-session.test.ts

```diff
@@ -126,6 +129,28 @@ describe('GenesysCloudMediaSession', () => {
       expect(spy).toHaveBeenCalledWith(JingleAction.SessionInfo, { info: { infoType: JINGLE_INFO_ACTIVE } });
     });
 
+    it('should call setupDataChannel when connected', () => {
+      const parent = new FakeParent();
+
+      const session = new GenesysCloudMediaSession({
+        options: { parent } as any,
+        sessionType: 'softphone',
```

Comment:
I get the impression we're mostly interested in data channels for video. Is that significant enough that it should be restricted by `sessionType`?

Reply:
I think I should clarify this. I wondered if there was any danger in a data channel being offered for a `sessionType` that we didn't expect (since we create one if it's been offered)? (In thinking a bit more, I think the danger is low since we're just emitting an event for clients to listen to.)

Reply:
answered via chat. tldr; it doesn't matter in this case.